testcase/oom/tikv_test.go

```diff
@@ -11,12 +11,12 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
-	qe_metrics "github.com/PingCAP-QE/metrics-checker/pkg/metrics"
 	_ "github.com/go-sql-driver/mysql"
+	infraresource "github.com/pingcap/test-infra/sdk/resource"
```

Comment:
IMO, `infra_resource` has better readability 🤔.

Reply:
but will make GoLand unhappy 😂😂

Reply:
I see😂. I checked the blog [package-names](https://blog.golang.org/package-names).

Maybe we can use name `resource` directly?